### PR TITLE
Refer MAC address and Generate MAC to each other

### PR DESCRIPTION
**Pull request by tagawa**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [X] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [X] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

After a discussion with @GuiltyDolphin and @edgesince84, this is a suggested change to highlight similar Instant Answers the user might be interested in. Both IAs currently use the text template meaning space is available for an infobox for this purpose, similar to the [Caesar cipher IA](https://duckduckgo.com/?q=how+to+use+a+caesar+cipher&ia=answer).

###### People to notify (@mention interested parties)
@GuiltyDolphin @moollaza 

---

https://duck.co/ia/view/generate_mac
https://duck.co/ia/view/mac_address

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Goodie/GenerateMAC.pm [hunk 1]
```diff
@@ -23,10 +37,11 @@ handle remainder => sub {
     return "Here's a random MAC address: $address", structured_answer => {
         data => {
             title => $address,
-            subtitle => 'Random MAC Address'
+            description => 'Random MAC Address',
+            infoboxData => $infobox
         },
         templates => {
-            group => 'text'
+            group => 'info'
```
Comment (GuiltyDolphin) [inline lib/DDG/Goodie/GenerateMAC.pm]: We can just use `text` (`infoboxes` work fine regardless).

**Comment by tagawa** [comment]
Hi @moollaza. This is a suggested change adding an infobox to show related queries to increase IA discoverability. This PR is just for our two MAC address IAs but if it looks successful, it could be extended to others. Currently works on beta: https://beta.duckduckgo.com/?q=please%20generate%20mac%20address
What do you think?

**Comment by moollaza** [comment]
I know previously we experimented with "Related Queries" (on the old site design) and they were essentially never clicked. However, we do have "Related Topics" for Fatheads, but I think that's a little different.

This is an interesting idea though, however I don't think it should come from us manually modifying individual IAs -- or is that just to build a POC -- I imagine we could build this dynamically for IAs from the metadata, so that the code doesn't require any changes.

I think you should propose this to @russellholt and the rest of the team to get their opinions.

Showing this info every time the IA displays could be considered cluttering it...

**Comment by tagawa** [comment]
Thanks for the comments @moollaza. Yes, in practice this shouldn't be done manually. I was thinking of just trying out this one for MAC addresses as a test, to see what the effect is over a month for example.

@GuiltyDolphin has some example code showing how it could be done dynamically using metadata:
https://github.com/duckduckgo/duckduckgo/commit/e3ce1977495da3ae40494fa14b385ce8d38ff638

I might show it as a demo at the next meeting to see what the team thinks.

**Comment by tagawa** [comment]
Got no objections from the team so merging this to see how it goes.

**Pull request merged**